WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationStepView.swift

```diff
@@ -71,13 +78,17 @@ class MigrationStepView: UIView {
 
     override func layoutSubviews() {
         super.layoutSubviews()
-        mainScrollView.contentInset.bottom = actionsView.frame.size.height + Constants.bottomMargin
+        let bottomInset = actionsView.frame.size.height - safeAreaInsets.bottom
+        mainScrollView.contentInset.bottom = bottomInset + Constants.bottomMargin
+        mainScrollView.verticalScrollIndicatorInsets.bottom = bottomInset
+        mainScrollView.contentInset.top = Constants.topMargin
     }
 
     private enum Constants {
         static let centerContentMargins = UIEdgeInsets(top: 0, left: 30, bottom: 0, right: 30)
         static let stackViewSpacing: CGFloat = 20
-        static let bottomMargin: CGFloat = 20
+        static let bottomMargin: CGFloat = 10
+        static let topMargin: CGFloat = UINavigationBar().intrinsicContentSize.height
```

Comment:
The `topMargin` constant increases the top inset of the Notifications and Success scrollViews, so that those screens look similar to the Welcome Screen.

The top inset in the Welcome Screen is set automatically because of the Navigation Bar, but since the Navigation Bar is hidden in the Notifications & Success screen, we have to increase scrollView the top inset manually.

But I'm not sure if the value of this `topMargin` should be `UINavigationBar().intrinsicContentSize.height`. 

Another approach would be to inject the `navigationController?.navigationBar` instance into this view and use its `intrinsicContentSize.height` property.

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/19632/commits/cbc74ab74b07d1a814a7b2c86e78b5d45572598e